src/main/java/teammates/ui/controller/InstructorCourseEnrollSaveAction.java

```diff
@@ -55,7 +55,9 @@ public ActionResult execute() throws EntityDoesNotExistException {
 
             return createShowPageResult(Const.ViewURIs.INSTRUCTOR_COURSE_ENROLL_RESULT, pageData);
 
-        } catch (EnrollException | InvalidParametersException e) {
+        } catch (EnrollException e) {
+
+        } catch (InvalidParametersException e) {
```

Comment:
and why are they presenting the error message differently? Why were they not splitted prior to this PR?

Reply:
Prior to this PR, exception messages are added through `setStatusForException` which resides in `Action`. A page refresh is then done and then error messages are shown through the usual status boxes present in most parts of TEAMMATES:
![image](https://user-images.githubusercontent.com/25320782/43816283-6fa8c758-9b06-11e8-8936-f333d85b29fd.png)

Since the discussion from #8989, the consensus is to use AJAX to display the necessary
feedback message on the current enroll page. As such, EnrollException messages will be thrown to facilitate the different types of messages shown in the enrollment result page:

![image](https://user-images.githubusercontent.com/25320782/43816433-5c2f128a-9b07-11e8-92a4-5a15a1af2241.png)
The following screenshot shows the changes this PR made:
![image](https://user-images.githubusercontent.com/25320782/43816356-d6202cb0-9b06-11e8-8223-375650f9bc26.png)
- Green corresponds to new students added.
- Yellow corresponds to students who are modified.
- Blue corresponds to students who are updated with no changes (aka enrolled again with same details).
I didn't show unmodified students here because that is basically the rest of the students in the course.

Therefore I separated these two exceptions as they will be showing the results differently. `InvalidParametersException` will still be throwing status message similar to the original version.

P.S Errors will be shown in red like below:
![image](https://user-images.githubusercontent.com/25320782/43816565-f5b99de4-9b07-11e8-856f-bd904d045c4a.png)

Reply:
This explain what you want to do but not why you have allowed user to edit current students from this spreadsheet. I don't agree with the fact that the name of the spreadsheet is "New students", yet you can also edit the details of existing students.

Reply:
This is not a new behavior I'm introducing but rather something that has existed previously. If the user enrolls a student with the same email but different details, they are treated as being modified (assuming no errors) and no new student would be created.

I agree that the name of the spreadsheet might not reflect this behavior but is this a naming issue or an implementation issue? Currently there is no way to prevent users from performing this behavior and not treating it as an "edit".

Reply:
I see. If this is the existing behaviour, then let's keep it like that and leave the problem of questioning whether that is a good behaviour in a different issue and PR.